### ISSUE TITLE
Missing logmessage if signalduino_protocols.hash is missing

### ISSUE THREAD

**Issue by sidey79** [RFD-FHEM/RFFHEM#516]
## Expected Behavior

If the protocol hash could not be loaded, then a meaningfull log messages is needed. 

There is already one in the code:
`Error loading Protocol Hash. Module is in inoperable mode error message`

## Actual Behavior

There is not really a hint, that the hash could not be loaded:

```
2019.02.21 21:35:56.710 0: Featurelevel: 5.9
2019.02.21 21:35:56.712 0: Server started with 651 defined entities (fhem.pl:18623/2019-02-17 perl:5.024001 os:linux user:fhem pid:10904)
2019.02.21 21:35:57.235 3: sduino IdList: ### Attribute development is in this version ignored ###
2019.02.21 21:35:57.256 3: sduino: IDlist attr whitelist disabled or not defined (all IDs are enabled, except blacklisted and instable IDs):
2019.02.21 21:35:57.275 3: sduino: IDlist MS
2019.02.21 21:35:57.295 3: sduino: IDlist MU
2019.02.21 21:35:57.314 3: sduino: IDlist MC

```
## Steps to Reproduce the Problem

  1. delete or rename signalduino_protocols.hash file
  2. restart fhem
  3.  look at log messages that the signalduino_protocols.hash could not be loaded

## Specifications

  - Versionmodul (FHEM Module): 3.3.3

signalduino_protocols

**Comment by sidey79** [comment]
[GitMate.io](https://gitmate.io) thinks possibly related issues are https://github.com/RFD-FHEM/RFFHEM/pull/512 (Update signalduino_protocols.hash), https://github.com/RFD-FHEM/RFFHEM/issues/346 (Missing length_max check at in SIGNALduino_OSPIR), https://github.com/RFD-FHEM/RFFHEM/issues/347 (Missing length_max check at in SIGNALduino_MCTFA ), and https://github.com/RFD-FHEM/RFFHEM/issues/348 (Missing length_max check at in SIGNALduino_Maverick).

**Comment by sidey79** [comment]
fixed via #523
